intranethome/views.py

```diff
@@ -0,0 +1,254 @@
+        data_from_file = json.loads(json_text)
+        return data_from_file
+
+# purpose of function: helper function for get_list_of_aliases to move numbers to the end
+# input: the list of aliases in the original order with numbers in front
+# output: the re-ordered list of aliases
+def helper_function_order_number_aliases_last(all_aliases):
+    list_of_numbers = []
+
+    for alias in all_aliases:
+        if alias[0] in "0123456789":
+            list_of_numbers.append(alias)
+
+    for number in list_of_numbers:
+        all_aliases.remove(number)
+        all_aliases.append(number)
+
+    return all_aliases
+
+# purpose of function: pulling the aliases of the data and creating an organized list
+# input: the python data from the file
+# output: a sorted list of the aliases
+def get_list_of_sorted_aliases(data_from_file):
+    aliases = list({ k for (k,v) in data_from_file.items() })
+    aliases.sort(key=str.lower)
+
+    aliases = helper_function_order_number_aliases_last(aliases)
+
+    return aliases
+
+# purpose of function: filtering the alias values to only return a subset of them based on user input
+# input: the full list of aliases and the value which they are filtered by
+# output: a subset of the aliases based on the filter
+def filter_by_value(original_alias_list, filter_value): 
+    filtered_alias_list = []
+    
+    for alias in original_alias_list:
+        
+        # if the filter is number
+        if filter_value == "number":
+            if alias[0] in "0123456789":
+                filtered_alias_list.append(alias)
+
+        # if the filter is a single letter
+        elif filter_value != "":
+            if alias[0] == filter_value:
+                filtered_alias_list.append(alias)
+
+        # if the filter is nothing, indicating that all values should be displayed
+        else:
+            return original_alias_list
+        
+    return filtered_alias_list
+
+# purpose of function: formatting the different types of data into lists of dictionaries
+# input: each individual dictionary
+# output: a formatted dictionary
+def uniforming_into_list_of_dict(note_or_email):
+    
+    # option 1: single key dictionary with key = 'email' and value with a local or gobal email
+    #   {'email': 'postmaster'}
+
+    # option 2: list of single dictionaries with  key = 'email' and value with a single local or global email, in one of 3 formats
+    #   [{'email': 'lhauglan@midway.uchicago.edu'}, ..., {'email': 'd-larsen@uchicago.edu'}]
+    #   [{'email': 'fierrom@uchicago.edu (Maria Fierro)'}, ..., {'email': 'hartj@uchicago.edu (Jenny Hart)'}]
+    #   [{'email': 'Ariel Erbacher\t\t<aaescotese@uchicago.edu>'}, ..., {'email': 'Amy Boucher\t\t<aebouch@emory.edu>'}]
+
+    # option 3: dictionary with key = 'note'
+    #   {'note': 'emails go to command: "|/usr/local/mailman/mail/mailman post access-ip'}
+
+    # selecting for option 1 or 3 in order to put them into both into lists
+        if(type(note_or_email) == dict):
```

Comment:
This blurb is over indented. Also, there are lots of general linting errors. All the `.py` files need to be linted with `Black` and all our other fixers`. If you're not setup with that, you can do that in Vagrant with Vim. More details under "Linting with VIM inside the Vagrant environment" here: https://github.com/uchicago-library/library_website/pull/641